perfkitbenchmarker/vm_util.py

```diff
@@ -90,6 +88,33 @@ class IpAddressSubset(object):
                   'external IP only (EXTERNAL) or internal IP only (INTERNAL)')
 
 
+class FakeFlagValues(object):
```

Comment:
This doesn't seem like a great place for this class. Maybe introduce a module for current context?

Also (nit): How is this Fake? Maybe `FlagValuesProxy`?